integrations/azure/README.md

```diff
@@ -0,0 +1,71 @@
+# Azure ORT Starter Pipeline
+
+The starter pipeline allows you to easily run an ORT scan on [Azure DevOps](https://azure.microsoft.com/services/devops/).
+
+To run the pipeline, simply:
+
+- Create a new YAML-based pipeline in your Azure DevOps project based on [ort-pipeline.yml](./ort-pipeline.yml) in this repository
+- Run the pipeline with the preferred set of parameters
+
+## Directory description
+
+This subdirectory has the following structure:
+```bash
+azure
+├── credentials
+│   ├── write-config-netrc.yml
+│   └── write-downloader-netrc.yml
+├── docker
+│   ├── docker-build-image.yml
+│   └── docker-ort-run.yml
+├── ort-pipeline.yml
+├── README.md
+└── steps
+    └── complete-ort-run.yml
+```
+The folder `credentials` contains templates to write `.netrc` files:
+- `write-config-netrc.yml` converts the parameter group  `ort_config_credentials` to `.netrc` used when downloading the configuration
```

Comment:
Maybe put a reference to the .netrc specification ?
https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html